src/modules/wallet.js

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
Can you fix the identing in this file?

Reply:
Fixed on https://github.com/HathorNetwork/hathor-wallet/pull/580/commits/81d1e27384ab27f4bd14dda7e9a0c719230112d4